app/code/Magento/Checkout/Test/Mftf/Test/StorefrontCheckoutVerifyAuthenticatePopupNotVisible.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCheckoutVerifyAuthenticatePopupNotVisible">
+        <annotations>
+            <features value="Checkout"/>
+            <stories value="Authentication popup should be removed from Checkout page"/>
+            <title value="Verify authenticate popup not visible in checkout page"/>
+            <description value="Verify authenticate popup not visible in checkout page"/>
+            <severity value="AVERAGE"/>
+            <testCaseId value="MC-36218"/>
+            <group value="checkout"/>
+        </annotations>
+
+        <before>
+            <!--Create Simple Products-->
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="SimpleProduct" stepKey="createSimpleProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+        </before>
+        <after>
+            <!--Delete created data-->
+            <deleteData createDataKey="createSimpleProduct" stepKey="deleteSimpleProduct"/>
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+        </after>
+
+        <!--Go to product page-->
+        <amOnPage url="$$createSimpleProduct.custom_attributes[url_key]$$.html" stepKey="navigateToSimpleProductPage"/>
```

Comment:
Great job. The last thing I would ask you to do is to utilize the existing action groups for such actions. 

https://github.com/magento/magento2/blob/0262f1428561f5c329a50bfbd62714fa58e175be/app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontOpenProductPageActionGroup.xml#L10

Thank you!